swo_aws_extension/flows/steps/terminate.py

```
@@ -30,7 +34,7 @@
 logger = logging.getLogger(__name__)
 
 
-class TerminateResponsibilityTransferStep(BasePhaseStep):
+class TerminateResponsibilityTransferStep(BasePhaseStep):  # noqa: WPS214
```

Ruff (0.16.1)

[warning] Invalid rule code in `# noqa`: WPS214

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)